lua/telescope/builtin/files.lua

```diff
@@ -145,106 +145,126 @@ files.grep_string = function(opts)
   }):find()
 end
 
--- TODO: Maybe just change this to `find`.
--- TODO: Support `find` and maybe let people do other stuff with it as well.
-files.find_files = function(opts)
+local function which_find_command(opts)
+  local command = opts.find_command
+  if command then
+    return command[1]
+  end
```

Comment:
Is this a good idea?

Are there cases when the first element of the table is not the command?

Reply:
What if someone implemented his own find implementation and uses this as `find_command`. Then we will return nil.

Reply:
Then its not handled by the `find_command_options` if ?

Reply:
The `which_find_command` function is just to know the options to apply depending on which of these `fd, fdfind, rg, find or where` are using, if someone use a custom `find_command` generally `(idk if it's always the case)` the first element on that list will be the command

Reply:
if that custom command is not one of the mentioned, then it will bypass all the options `(all the if statements in find_command_options)` and return just the `find_command` the user set

Reply:
yep